src/main/java/com/fastmail/jstpierre/petmanager/services/PetOwnerService.java

```diff
@@ -123,7 +127,40 @@ public void changePetOwnerAddressByPetId(String newPetOwnerAddress, String petId
         return petOwner.getPetToys();
     }
 
-    // TODO Add method to calculate total price paid for owner toys
+    /** ex.: { toys: [{name: abc, pricePaid: 3.99}, {name: def, pricePaid: 1.99}], total: 5.98} */
+    public HashMap<String, Object> getPetToysPurchaseDetailByHousehold(PetOwner petOwner) {
+        if (petOwner.getPetToys().isEmpty()) {
+            return null;
+        }
+
+        final String KEY_NAME = "name";
+        final String KEY_PRICE_PAID = "pricePaid";
+        final String KEY_TOTAL = "total";
+        final String KEY_TOYS = "toys";
```

Comment:
Normalement, on réserve les variables en all-caps pour des constantes de classe.
Comme ici, ce sont des variables immutables dans une méthode, il faut conserver le camel case.
```suggestion
        final String keyName= "name";
        final String keyPricePaid = "pricePaid";
        final String keyTotal = "total";
        final String keyToys= "toys";
```